treeherder/config/settings.py

```diff
@@ -142,6 +143,14 @@
     'default': env.db_url('DATABASE_URL', default=LOCALHOST_MYSQL_HOST),
 }
 
+SKIP_INGESTION = env('SKIP_INGESTION', default=False)
+
+# Block for changes within the context of local development (venv & Docker)
+if not PRODUCTION:
+    # Do not permit ingesting if DATABASE_URL points to a database different than localhost
+    if DATABASES['default']['HOST'] not in ['mysql', '127.0.0.1', 'localhost']:
```

Comment:
Not sure if this is entirely needed since devs should be using the production replica for this.

Reply:
The idea is that if we're running Treeherder locally and DATABASE_URL does not match any valid local DB (`DATABASES['default']['HOST']` is derived from DATABASE_URL) then we should not ingest.